labelbox/schema/identifiable.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Union
+
+
+class IdType(str, Enum):
+    DataRowId = "UID"
```

Comment:
in ADV these are ID and GKEY, these are used by the census / bucket scanning APIS.

Reply:
ok I can change it, but in a follow up PR because this will require the api update too